remote_attestation/java/src/com/google/oak/remote_attestation/crypto/tink/AeadEncryptor.java

```diff
@@ -0,0 +1,39 @@
+package com.google.oak.remote_attestation.crypto;
```

Comment:
Specifically here it is temporary, because since we currently have 2 different implementations, we need to do conditional compilation (that Java doesn't support, but it can be done via Bazel).